### PR TITLE
eslint fixes and standardize exports style

### PR DESCRIPTION
**Pull request by isonmad**
Put exports in one place per file and use the same style in each one.

Along the way as I tried editing run-web-platform-tests.js, I found out it had lint errors.

### CODE REVIEW

**Comment by ricea** [comment]
lgtm, thanks.

**Comment by domenic** [comment]
Can you update the .eslintignore patterns and such to ensure eslint is actually applied to the files you're fixing, and thus fails the build?

File: reference-implementation/.eslintrc.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     // Possible errors
     "comma-dangle": ["error", "never"],
     "no-cond-assign": ["error", "except-parens"],
-    "no-console": "error",
+    "no-console": "off",
```
Comment (domenic) [inline reference-implementation/.eslintrc.json]: Don't turn this off globally. Turn it off inside the file that uses console, using `/*eslint-disable no-console */`

File: reference-implementation/run-web-platform-tests.js [hunk 2]
```diff
@@ -21,11 +21,11 @@ wptRunner(toUpstreamTestsPath, { rootURL: 'streams/', setup })
   })
   .then(failures => {
     totalFailures += failures;
-    process.exit(totalFailures);
+    process.exitCode = totalFailures;
```
Comment (domenic) [inline reference-implementation/run-web-platform-tests.js]: Don't change this just to get around the restriction. Use `/*eslint-disable*/`

File: reference-implementation/.eslintrc.json (hunk 1 above)
Comment (isonmad) [inline reference-implementation/.eslintrc.json]: Ah, I wasn't familiar enough with eslint to know I could do that. Good idea.

File: reference-implementation/run-web-platform-tests.js (hunk 2 above)
Comment (isonmad) [inline reference-implementation/run-web-platform-tests.js]: Wait, why not? Why would the process need to exit immediately, when it's about to exit on its own naturally regardless as this is the last thing that gets run and nothing else is keeping process alive? This seems cleaner.

Comment (domenic) [inline reference-implementation/run-web-platform-tests.js]: I guess that's true; thanks for pushing back.

**Comment by isonmad** [comment]
> Can you update the .eslintignore patterns and such to ensure eslint is actually applied to the files you're fixing, and thus fails the build?

I was trying to figure out what this meant, since there are no .eslintignore files anywhere in the repo. But now I see the problem, it's the wildcards in lib/package.json.

**Comment by isonmad** [comment]
Okay, I moved eslint to a blacklist of things that shouldn't be linted, instead of a whitelist of things that should; is there anything else blocking this being merged now?

**Pull request merged**